bgpd/bgp_zebra.c

```diff
@@ -1545,6 +1545,11 @@ void bgp_zebra_withdraw(struct prefix *p, struct bgp_path_info *info,
 	api.safi = safi;
 	api.prefix = *p;
 
+	if (info->attr->rmap_table_id) {
```

Comment:
The description of this bug is not consistent with what this is doing.  Let's get that fixed up.

Reply:
bgpd: withdraw fib entry on appropriate table identifier
    
    There are cases where the table identifier is set on a bgp entry, mainly
    due to route-map, and associate fib entry needs to be removed.
    This change encompasses also the route-map reconfiguration that leads to
    removing the previous entry, whereas bgp update had been triggered (
    this happens when software inbound reconfiguration is handled).